File: strings/2390_removing_stars_from_a_string.go
func removeStars(s string) string {

    stack := []rune{}
    ans := ""

    for _, r := range s {

        if r != '*' {
            stack = append(stack, r)
        } else if len(stack) > 0 {
            stack = stack[:len(stack)-1]
        }
    }

    ans = string(stack)

    return ans

}

File: strings/2390_removing_stars_from_a_string.js
/**
 * @param {string} s
 * @return {string}
 */
var removeStars = function(s) {

    const stack = []

    for(l of s) {
        if(l === "*") stack.pop()
        else stack.push(l)
    }

    return stack.join("")
};